release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -85,6 +85,28 @@ CRI-O brings:
 
 image::crio-3-7.png[CRI-O]
 
+To install and run CRI-O alongside `docker`, set the following in the
+`[OSEv3:vars]` section
+xref:../install_config/install/advanced_install.adoc#configuring-ansible[Ansible inventory file] during cluster installation:
+
+----
+openshift_use_crio=True
+----
+
+This setting pulls the *openshift3/cri-o:latest* image from the link:https://access.redhat.com/containers[Red Hat Registry]. It also sets
```

Comment:
It's not latest on the enterprise install. It will use the ``openshift_image_tag`` which is sensed in the installer or set in the inventory.

Reply:
OK, will just drop `:latest` here.